src/main/java/uk/gov/hmcts/ccd/v2/external/controller/CaseController.java

```diff
@@ -211,4 +219,43 @@ public CaseController(
 
         return status(HttpStatus.CREATED).body(new CaseResource(caseDetails, content, ignoreWarning));
     }
+
+
+    @GetMapping(
+        path = "/cases/{caseId}/events",
+        headers = {
+            V2.EXPERIMENTAL_HEADER
+        },
+        produces = {
+            V2.MediaType.CASE_EVENTS
+        }
+    )
+    @ApiOperation(
+        value = "Retrieve an event by case and event IDs for dynamic display",
+        notes = V2.EXPERIMENTAL_WARNING
+    )
+    @ApiResponses({
+        @ApiResponse(
+            code = 200,
+            message = "Success",
+            response = CaseEventsResource.class
+        ),
+        @ApiResponse(
+            code = 400,
+            message = ERROR_CASE_ID_INVALID
+        ),
+        @ApiResponse(
+            code = 404,
+            message = "Case event not found"
```

Comment:
Should it not say - Case audit events not found?

Reply:
Changed